Update web3js tutorial (#5898)
ethereum/ethereum-org-website: src/content/developers/tutorials/set-up-web3js-to-use-ethereum-in-javascript/index.md

getBlockNumber()
```

> You can see all the functions that are available on the Web3 instance in the [official Web3 documentation](https://web3js.readthedocs.io/en/v1.2.6/web3-eth.html#).

Reviewer: ```suggestion
You can see all the functions available on the Web3 instance in [the official web3.js documentation](https://web3js.readthedocs.io/en/v1.2.6/web3-eth.html#).
```
